warehouse/models/mart/ntd/_mart_ntd.yml

```diff
@@ -0,0 +1,33 @@
+version: 2
+
+models:
+  - name: dim_annual_database_agency_information
+    tests:
```

Comment:
Please add a description that explicitly identifies what year / table data this is, preferably with a pretty specific link to the source data and perhaps a note about how to join with organizations.